composer.json

```diff
@@ -25,13 +25,14 @@
         "bref/bref": "^0.5.18",
         "async-aws/sns": "^1.0",
         "async-aws/sqs": "^1.2",
-        "async-aws/event-bridge": "^1.0"
+        "async-aws/event-bridge": "^1.0",
+        "symfony/amazon-sqs-messenger": "5.2.x-dev"
```

Comment:
I will update this when 5.2 is released (or is there another way im not aware of?)